.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "papaya-interface"]
```

Comment:
Dodałbyś tutaj wzmiankę o branchu (chyba można `branch = master`)?. 

Generalnie da się zmienić brancha w submodule, nie? Opisałbyś jak? Wygodniej, jak się chce coś dodać testowo, żeby zobaczyć, czy działa, albo jakby coś czekało na zmerge'owanie na mastera

Reply:
Pliki można edytować nawet lokalnie i je commitować z submodułu (do repozytorium papaya-interface). Zmiany gałęzi wyglądają identycznie jak w zwykłym repozytorium. 

Dodałem to branch = master (ale chyba to jest domyślne).

Działa to tak, że jeśli przy commicie frontowym submoduł był na jakimś konkretnym commicie z mastera interface, to nawet jeśli pojawiłyby się w interface nowe commity, to i tak przy `submodule update` wybrany będzie zawsze ten sam (ten nie najnowszy). 
Żeby wybrać "ten najnowszy", trzeba zrobić pulla w submodule. Wtedy przy commitowaniu frontu pojawi się tajemniczy plik `papaya-interface` (którego zawartości z jakiegoś powodu nie da się wyświetlić w intelliJ) i to właśnie on będzie informacją "ej teraz submoduł ma być domyślnie na tym commicie".